package.json

```diff
@@ -120,5 +120,8 @@
       "pre-commit": "npm run test:eslint",
       "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
     }
-  }
+  },
+  "browserslist": [
```

Comment:
Is this necessary because we don't have the latest version of babel? I know we had a warning for `browserlist` when we ran the tests in the other repos, but it went away as I upgraded all of our dev dependencies there.
I haven't had a chance to get to this repo yet

Reply:
Yep, otherwise `npm run serve` does not work. That's why I mentioned it directly in the PR description.